DESCRIPTION

Cloning should fail when destination directory exists and is not empty

In msysgit, when I try to clone into a folder that already
contains a cloned repository it will fail with a message "fatal:
destination path 'folder' already exists and is not an empty
directory.". Now JGit throws a JGitInternalException if the folder to
clone into exists and is not empty.

Test case provided by Clay Cameron.

Bug: 347852
Change-Id: If9e9919a5f92d13cf038dc470c21ee5967322dac
Also-by: Clay Cameron xxx@xxx.xxx
Signed-off-by: Ximena Pugh xxx@xxx.xxx


COMMENTS

author: Ximena Pugh
date: 2011-07-05 14:21:55.000000000

Uploaded patch set 4.

-------------------------------------
author: Brenden Conley
date: 2011-07-05 14:21:58.000000000

Patch Set 4:

Build Started https://hudson.eclipse.org/sandbox/job/jgit.gerrit/623/ 

-------------------------------------
author: Brenden Conley
date: 2011-07-05 14:25:34.000000000

Patch Set 4: Build Successful

Build Successful 
 
https://hudson.eclipse.org/sandbox/job/jgit.gerrit/623/ : SUCCESS

-------------------------------------
